.gitignore

```diff
@@ -0,0 +1,18 @@
+relx
```

Comment:
It's the generic .gitignore we use at @inaka for all erlang projects, I can trim it down.